### PR TITLE
Tag XSim v0.1.0

### PR DESCRIPTION
**Pull request by reworkhow**

### CODE REVIEW

File: XSim/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.4
+DataFrames
+Distributions
+JWAS
```
Comment (tkelman) [inline XSim/versions/0.1.0/requires]: does this depend on the latest version?

**Comment by tkelman** [comment]
you should consider adding some real tests to https://github.com/reworkhow/XSim.jl/blob/v0.1.0/test/runtests.jl

File: XSim/versions/0.1.0/requires (hunk 1 above)
Comment (reworkhow) [inline XSim/versions/0.1.0/requires]: It needs latest JWAS

**Comment by reworkhow** [comment]
real tests have been added

File: XSim/versions/0.1.0/requires (hunk 1 above)
Comment (tkelman) [inline XSim/versions/0.1.0/requires]: then you should add a minimum version requirement here

File: XSim/versions/0.1.0/requires [hunk 2]
```diff
@@ -1,4 +1,4 @@
 julia 0.4
 DataFrames
 Distributions
-JWAS
+JWAS 0.1-
```
Comment (tkelman) [inline XSim/versions/0.1.0/requires]: This is good now, but just so you're aware, the trailing `-` means "any prerelease of JWAS 0.1.0 or newer."

**Pull request merged**